.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -19,16 +19,14 @@ For a PR to be considered, each item in the checklist must be checked.
 <!-- To record and share a video: http://recordit.co/ -->
 
 ### How to test
-<!--
-Please describe the steps for reviewers to take to cover all facets of this feature
+<!-- Please describe the steps for reviewers to take to cover all facets of this feature. -->
 
-1. `npm run clean && npm install`
-2. navigate to Site > Button > Buttons with kittens section
-3. Click the Cat Button twice (three times in Internet Explorer)
+1. `cd mineral-ui/packages/button && npm start`
+2. Navigate to Site > Button > Buttons with kittens section
+3. Click the Cat Button twice (three times in IE 11)
 4. ¡Hear the cat meow en Español!
 
--->
-<!-- Please describe in detail how you tested your changes beyond automated tests. -->
+<!-- Is this a new component or a breaking change? Make sure you also spin up a sandbox (https://github.com/facebookincubator/create-react-app#getting-started), test it there, and tell us about it. -->
```

Comment:
Do you think this should be bigger? Maybe part of the checklist? I'm trying to verify that it can be done locally before releasing. I've only ever pulled our released components into create-react-app before...